general/administration/installing.md

```diff
@@ -63,25 +63,38 @@ An Unraid Docker template is available in the repository.
 
 A community project to deploy Jellyfin on Kubernetes-based platforms exists [at their repository](https://github.com/home-cluster/jellyfin-openshift). Any issues or feature requests related to deployment on Kubernetes-based platforms should be filed there.
 
-## Portable Binaries
+## Windows (x64)
```

Comment:
why only `x64`, did we drop 32-bit for Windows?

Reply:
There is no installer for x86. And I think we should drop it, nobody in their right mind should be using anything 32-bit, besides anything that is restricted to 32-bit will be to slow to run Jellyfin satisfactorily anyway.

Reply:
Can we note that 32-bit version is ZIP-only (and is deprecated but still available for now)?